plonk-core/src/constraint_system/composer.rs

```diff
@@ -17,11 +17,13 @@
 use crate::{constraint_system::Variable, permutation::Permutation};
 use alloc::collections::BTreeMap;
 
+use crate::lookup::LookupTable;
 use ark_ec::{models::TEModelParameters, ModelParameters};
 use ark_ff::PrimeField;
 use core::marker::PhantomData;
 use hashbrown::HashMap;
 use rand::{CryptoRng, RngCore};
+use std::cmp::max;
```

Comment:
```suggestion
use ark_std::cmp::max;
```

A `cargo fmt` will be needed after this change.

Reply:
Please use `core` for this.

Reply:
So in ZK-Garage we prefer core::/alloc:: instead of ark_std::? Just to follow the habits here.

Also in arkworks, we generally get rand:: from ark_std::rand:: to avoid version conflicts.

Reply:
We should just be using `rand_core` anyway, which has a very stable interface and is actually `no_std`.

Reply:
Switched to `core::cmp::max` and `rand` has been substituted for `rand_core`.